src/components/AddresseeForm.vue

```diff
@@ -24,6 +24,7 @@
         <div class="p-col-4 p-lg-2 p-text-bold">送貨方式</div>
         <div class="p-col-8 p-lg-10">
           <Dropdown
+            @change="$emit('shipping-method', shipping_info.shipping_method)"
```

Comment:
這邊可以使用下面寫的 method 就好
```suggestion
            @change="emitShippingMethod"
```

Reply:
改好了~
https://github.com/c711cat/lemon_coffee/pull/54/commits/e073593c2d24be62440f0070a8fb1e515dd59744